.github/workflows/main.yml

```diff
@@ -1,43 +1,42 @@
 name: Application testing
 on:
-  push:
-  workflow_dispatch: 
+    push:
+    workflow_dispatch:
 jobs:
-  unit-tests:
-    name: Unit tests
-    runs-on: self-hosted
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
-    
-      - name: Setup Node
-        uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-          
-      - name: Install yarn
-        run: npm install -g yarn
+    unit-tests:
+        name: Unit tests
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v2
 
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+            - name: Setup Node
+              uses: actions/setup-node@v1
+              with:
+                  node-version: "12.x"
 
-      - name: Cache yarn dependencies
-        uses: actions/cache@v2
-        id: yarn-cache
-        with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
-      - name: Installing Dependencies
-        run: yarn install --frozen-lockfile --silent
+            - name: Install yarn
+              run: npm install -g yarn
 
-      - name: Install translations
-        run: yarn localize
-      
-      - name: Run jest tests
-        run: yarn test
+            - name: Get yarn cache directory path
+              id: yarn-cache-dir-path
+              run: echo "::set-output name=dir::$(yarn cache dir)"
 
-      - name: Run typescript tests
-        run: npx tsc
+            - name: Cache yarn dependencies
+              uses: actions/cache@v2
+              id: yarn-cache
+              with:
+                  path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+                  key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+                  restore-keys: |
+                      ${{ runner.os }}-yarn-
+            - name: Installing Dependencies
+              run: yarn install --frozen-lockfile --silent
+
+            - name: Install translations
+              run: yarn localize
+
+            - name: Run jest tests
+              run: yarn test
+
+            - name: Run typescript tests
+              run: npx tsc
```

Comment:
Removed this line `runs-on: self-hosted` because we though that was generating the error:

```
This request was automatically failed because there were no enabled runners online to process the request for more than 1 days.
```

Looks like the error is not related to that setting @adrianq